### PR TITLE
REL: v1.9.0

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -1372,13 +1320,13 @@ authors:
   - given-names: Xiangrui
     family-names: Li
     website: https://github.com/xiangruili
-    affiliation: The Ohio State University, United States of America
```
Comment (Remi-Gau) [inline CITATION.cff:1375]: not sure why the country was removed here

File: CITATION.cff [hunk 2]
```diff
@@ -1372,13 +1320,13 @@ authors:
...
+    affiliation: The Ohio State University
     email: xiangrui.li@gmail.com
   - given-names: Yaroslav O.
     family-names: Halchenko
     website: http://www.onerussian.com
     orcid: https://orcid.org/0000-0003-3456-2493
-    affiliation: Dartmouth College, United States of America
```
Comment (Remi-Gau) [inline CITATION.cff:1381]: and here

File: CITATION.cff [hunk 3]
```diff
@@ -849,7 +822,7 @@ authors:
     family-names: Zehl
     website: https://github.com/lzehl
     orcid: https://orcid.org/0000-0002-5947-9939
-    affiliation: Forschungszentrum Jülich, Jülich, 52428, Germany
```
Comment (Remi-Gau) [inline CITATION.cff:852]: why was this shortened this much?

File: CITATION.cff
Comment (Remi-Gau) [inline CITATION.cff:1]: Note for the contributor workflow PR: more changes than needed here

**Comment by Remi-Gau** [comment]
Leaving some comments in the PR that are more related to the contributors update worflow: not important for the release per se.

**Comment by Remi-Gau** [comment]
@sappelhoff 

This section was a bit unclear:

https://github.com/bids-standard/bids-specification/blob/master/Release_Protocol.md#23-update-the-previous-version-urls

- should mention that this about the `src/CHNAGES.md`
- also either this changes has to be reverted after the release and it is not mentioned in the release protocol or this step was not applied in the previous release

**Comment by Remi-Gau** [comment]
OK we should mention both points in the protocol then.

**Comment by sappelhoff** [comment]
cc @sjeung @JuliusWelzel FYI we'll release in around 5 days :-)

**Review by effigies — APPROVED** [review]

**Review by sappelhoff — APPROVED** [review]

**Comment by Remi-Gau** [comment]
ok. everyone stand back!!! we are merging!!!

**Comment by Remi-Gau** [comment]
> > should mention that this about the src/CHANGES.md
> 
> You are right @Remi-Gau it should mention that.
> 
> > also either this changes has to be reverted after the release and it is not mentioned in the release protocol or this step was not applied in the previous release
> 
> it is automatically "overwritten" by the GitHub Changelog Generator that we are using -- these steps have always been applied, as can be verified by checking the stable changelog: https://bids-specification.readthedocs.io/en/stable/CHANGES.html

@sappelhoff maybe I missed something in the release protocols but when are the URLs in the changelog reverted to the github version rather than the RTD one?

**Comment by effigies** [comment]
Next PR merge will trigger the changelog generator.

**Comment by sappelhoff** [comment]
> @sappelhoff maybe I missed something in the release protocols but when are the URLs in the changelog reverted to the github version rather than the RTD one?

this happens automatically (via the GH changelog generator) once the first PR is merged via a "create merge commit" (so not squash and not rebase) method (because that's what triggers the generator), see: https://github.com/bids-standard/bids-specification/blob/b0e6744421f3ffd2541fb49df5a5357652afaa8f/.circleci/config.yml#L110

**Comment by Remi-Gau** [comment]
> > @sappelhoff maybe I missed something in the release protocols but when are the URLs in the changelog reverted to the github version rather than the RTD one?
> 
> this happens automatically (via the GH changelog generator) once the first PR is merged via a "create merge commit" (so not squash and not rebase) method (because that's what triggers the generator), see:
> 
> https://github.com/bids-standard/bids-specification/blob/b0e6744421f3ffd2541fb49df5a5357652afaa8f/.circleci/config.yml#L110

ha OK then I messed that up...

**Comment by Remi-Gau** [comment]
https://github.com/bids-standard/bids-specification/commit/7bff0978b4e70f631cc00fb3c52aea7ecc94b128

The release protocol said commit to `master`, so committed AND pushed to master (on upstream) #YOLO

Probably worth fixing that in a separate PR.

**Comment by sappelhoff** [comment]
yes, together with the other items of note to change in the release protocol 👍

**Comment by Remi-Gau** [comment]
definitely a good thing I did this release then: allowed to clarify a few things

**Pull request merged**